README.md

```diff
@@ -53,7 +53,32 @@ const ws = new WS(properties)
 | upgrader | [`Upgrader`](https://github.com/libp2p/interface-transport#upgrader) | connection upgrader object with `upgradeOutbound` and `upgradeInbound` | **REQUIRED** |
 | filter | `(multiaddrs: Array<Multiaddr>) => Array<Multiaddr>` | override transport addresses filter | **Browser:** DNS+WSS multiaddrs / **Node.js:** DNS+{WS, WSS} multiaddrs |
 
-### Base Example
+## Libp2p Usage Example
+
+```js
+const Libp2p = require('libp2p')
+const Websockets = require('libp2p-websockets')
+const filters = require('libp2p-websockets/src/filters')
+
+const transportKey = Websockets.prototype[Symbol.toStringTag]
+const node = await Libp2p.create({
+  modules: {
+    transport: [WebRTCStar],
+    // ... other mandatory modules
```

Comment:
I would just add NOISE and mplex in here. There aren't other options right now, so having a config people can just use would be helpful imo.